bin/generate-php-sync-issue.mjs

```diff
@@ -0,0 +1,433 @@
+/**
+ * External dependencies
+ */
+
+import Octokit from '@octokit/rest';
+import fs from 'fs';
+
+import { fileURLToPath } from 'url';
+import nodePath, { dirname } from 'path';
+
+function getArg( argName ) {
+	const arg = process.argv.find( ( arg ) =>
+		arg.startsWith( `--${ argName }=` )
+	);
+	return arg ? arg.split( '=' )[ 1 ] : null;
+}
+
+const OWNER = 'wordpress';
+const REPO = 'gutenberg';
+
+const IGNORED_PATHS = [
+	'lib/experiments-page.php',
+	'packages/e2e-tests/plugins',
+	'packages/block-library', // this is handled automatically.
+];
+
+const DEBUG = !! getArg( 'debug' );
+
+const __filename = fileURLToPath( import.meta.url );
+const __dirname = dirname( __filename );
+
+const authToken = getArg( 'token' );
+
+async function main() {
```

Comment:
If folks agree with the premise then some tests for this function would be helpful.